Tests/ApolloSQLiteTests/CachePersistenceTests.swift

```diff
@@ -68,18 +78,43 @@ class CachePersistenceTests: XCTestCase {
       let networkExpectation = self.expectation(description: "Fetching query from network")
       let emptyCacheExpectation = self.expectation(description: "Fetch query from empty cache")
 
-      client.fetch(query: query, cachePolicy: .fetchIgnoringCacheData) { (result, error) in
+      client.fetch(query: query, cachePolicy: .fetchIgnoringCacheData) { outerResult in
         defer { networkExpectation.fulfill() }
-        guard let result = result else { XCTFail("No query result");  return }
-        XCTAssertEqual(result.data?.hero?.name, "Luke Skywalker")
+        
+        switch outerResult {
+        case .failure(let error):
+          XCTFail("Unexpected faillure: \(error)")
+        case .success(let graphQLResult):
+          XCTAssertEqual(graphQLResult.data?.hero?.name, "Luke Skywalker")
+        }
 
-        do { try client.clearCache().await() }
-        catch { XCTFail() }
+        do {
+          try client.clearCache().await()
+        } catch {
+          XCTFail("Error Clearing cache: \(error)")
+          emptyCacheExpectation.fulfill()
+          return
+        }
 
-        client.fetch(query: query, cachePolicy: .returnCacheDataDontFetch) { (result, error) in
+        client.fetch(query: query, cachePolicy: .returnCacheDataDontFetch) { innerResult in
```

Comment:
Why is this called `innerResult` here?

Reply:
Because this is a nested callback - we already have `outerResult` above.